app/helpers/content_helper.rb

```diff
@@ -45,7 +45,10 @@ def header(title, level)
       hash = header_anchor_hash(title)
 
       if @options[:header_anchors]
-        "<#{ tag } id=\"#{ hash }\"><a class=\"header_anchor\" href=\"\##{ hash }\">#{ title }</a></#{ tag }>"
+        "<#{ tag } id=\"#{ hash }\">" \
```

Comment:
I think in other places we don't escape the double quotes, we use single quotes instead. Similarly with the \ new line, I think it's nicer if it's just one big string.

Reply:
For the new lines, what would you suggest? I didn't want it to be a big single line, nor have extra spacing on the output.

Reply:
Here for example it's just a single string. What the output looks like doesn't matter as it all gets minified anyway.

https://github.com/Mitcheljager/workshop.codes/blob/b3c2db9db11f40a0e89c785c1d260f9582c24026/app/helpers/content_helper.rb#L85-L87